### ISSUE TITLE
Adopt the Integration Cookbook

### ISSUE THREAD

**Issue by rkratky** [fabric8io/fabric8-online-docs#116]
Convert, revise, and eventually publish the Integration Cookbook. See https://github.com/openshiftio/openshift.io/issues/1324

Currently available from https://docs.google.com/document/d/1k5hbknuebBZVVMkWpKZKAP6LoHWXD2bbvrn-4P4wmjg/

**Comment by Preeticp** [comment]
Subtasks:
- [ ] Review content post latest changes by Devs
- [ ] Convert to Asciidoc using Max's script
- [ ] Modularize
- [ ] Get a tech and peer review.

**Comment by Preeticp** [comment]
Currently on hold based on input from Developers. Will be pushed ahead at a later date, after receiving a go ahead from them.

**Comment by rkratky** [comment]
@Preeticp I have just merged #246, which created required dir structure for the cookbook (I used the title you suggested: Analytics Integration API Guide; if it turns out it cannot be used, I'll update the repo).

Let me know when you're done with review comments in #149, and I will then migrate the content to the new structure.

**Comment by rkratky** [comment]
@Preeticp Just merged #250, which prepped the landing page for the Cookbook. All is now ready and waiting on #116.

**Comment by Preeticp** [comment]
Completed by #255
